### ISSUE TITLE
The pretrained model and evaluation DRN

### ISSUE THREAD

**Issue by veroveroxie** [sapphire497/query-selected-attention#2]
Hi, thanks for your excellent work.

1. Could you please release the pre-trained models? 

2. Could you also please release your code of evaluation of cityscapes? The PixAcc of your method is super high, and I would like to compare and cite your work.

3. About the SWD, may I ask which code are you using? I tried google and I can only get https://github.com/koshian2/swd-pytorch . It uses tensors as input and there can be many ways of generating tensors from images. 

Again, thanks for your work! It is very interesting!

**Comment by sapphire497** [comment]
Hi!
1. The pre-trained models will be released in a few days.
2. I use the official codes in [https://github.com/fyu/drn](url), prepare your synthesis and label images, and run segment.py to calculate the pixAcc and mAP.
3. You can download the 'evaluation.zip' from [https://github.com/microsoft/CoCosNet/issues/25](url), and run cal_sliced_wasserstein.py to calculate the SWD.

**Comment by veroveroxie** [comment]
1. That would be very great! Thanks, looking forward to your models.
2. Could you please tell me which pretrained model do you use? the pretrained d-22 or d-105-ms? Did you resize the image and labels to 256x128 or 256x256 or 128x256?
3. Thanks!

**Comment by sapphire497** [comment]
I use d-22 model and resize the images to 256x128.

**Comment by veroveroxie** [comment]
Hi, I used your cat2dog checkpoint to generate images. I can get FID 80, which is the same of the paper. But I used SWD, 

I can only get following results

Horse2zebra
![image](https://user-images.githubusercontent.com/44474545/166091036-8438060a-c2d7-42c2-b686-fe50636ca8bc.png)

Cat2dog
![image](https://user-images.githubusercontent.com/44474545/166091019-72c84c44-b1eb-4794-a51a-749827438468.png)

Is it a typo in your paper? Or could you please tell me how to compute Swd for cat2dog?

**Comment by haoren55555** [comment]
Hi,could you please publish your pretrained segmentation model on cityscapes?
thanks!
